Assets/_Project/Code/Scripts/Summary/Panel.cs

```diff
@@ -38,6 +38,12 @@ public void Initialize(TeamManager team)
         float questions = goodAnswers + wrongAnswers;
         int cluesUsed = 5;
 
+        LeaderboardList leaderboard = new LeaderboardList();
+        leaderboard.Deserializuj();
+        leaderboard.AddTeam(new LeaderboardTeam(teamName, money));
+        leaderboard.Serializuj();
```

Comment:
Tutaj powinno spisywać TotalMoney. Przy okazji jakbyś mógł rozbudować tą drużynę jeszcze o odznaki i wyszukiwanie po nazwie drużyny to można będzie na niej już wszystko ogarnąć.